.github/workflows/docker-build.yml

```diff
@@ -0,0 +1,48 @@
+name: Docker Build
+
+on:
+  push:
+    branches: [ main ]
+    paths:
+      - 'Dockerfile'
+      - '.github/workflows/docker-build.yml'
+  pull_request:
+    branches: [ main ]
```

Comment:
This means, this Github Action with run if a pull request is made on main. You can see a history of runs in the “Actions” tab on github.